pkg/option/config.go

```diff
@@ -2509,6 +2484,47 @@ func (c *DaemonConfig) populateHostServicesProtos() error {
 	return nil
 }
 
+func (c *DaemonConfig) populateIdentityAllocation() error {
+	c.DisableCiliumEndpointCRD = viper.GetBool(DisableCiliumEndpointCRDName)
```

Comment:
How does CiliumEndpoint CRD relate to identities? ciliumidentities sure, but ciliumendpoint I'm not sure.